bin/remove_CatalogServiceZenPack.sh

```diff
@@ -18,7 +18,12 @@ if [[ ! $status -eq 0 ]]; then
     rm $tmp_output
     echo "Couldn't remove CatalogService zenpack"
     echo "You should remove CatalogService zenpack manually before starting the upgrade"
+    echo "To remove the CatalogService ZenPack:"
+    echo -e "1. Ensure the following services are running:\n\t * Zenoss.resmgr/Infrastructure/mariadb-model\n\t * Zenoss.resmgr/Infrastructure/mariadb-events\n\t * Zenoss.resmgr/Infrastructure/RabbitMQ\n\t * Zenoss.resmgr/Zenoss/Events/zeneventserver\n\t * Zenoss.resmgr/Infrastructure/redis\n\t * Zenoss.resmgr/Infrastructure/memcached"
+    echo -e "2. Ensure the following service has stopped:\n\t *Zenoss.resmgr/Infrastructure/zencatalogservice"
+    echo -e "3. Log onto the host the Control Center master and run the following command:\n\t serviced service run zope zenpack-manager uninstall ZenPacks.zenoss.CatalogService"
```

Comment:
-grin-
I sometimes leave words out when I write.  Please insert the word "of" between the words "host" and "the" in the phrase "Log onto the host the Control Center".